src/ui/server/controllers/insurance/eligibility/buyer-country/index.ts

```diff
@@ -115,10 +112,24 @@ export const post = async (req: Request, res: Response) => {
     }
 
     if (canApplyOffline(country)) {
```

Comment:
not a blocker.  Can this use `if (applyOnline)`

Reply:
It can't unfortunately, it needs to specifically check if it can apply offline which is different to online. It checks for this field from CIS data: `shortTermCover`:

It will only return true if the field has one of these values:
```js
ILC
CILC
REFER
```